README.md

```diff
@@ -20,49 +18,48 @@ If you want to install the PKC so that it is publicly accessible from the web:
-2. ` ./pkc setup --empty-wiki` will not pre-populate the wiki from https://github.com/inblockio/PKC-Content which includes default pages for how to use the PKC and other helpful ressources.
-3. ` ./pkc --nuke` is a command which deletes the mountpoint (the persistant data) and deletes the current instances of the docker containers.
+1. `./pkc setup --private` is setting the wiki into private mode by default. Except the main page all pages will not be visible to not registered users.
+2. `./pkc setup --empty-wiki` will not pre-populate the wiki from https://github.com/inblockio/PKC-Content which includes default pages for how to use the PKC and other helpful resources.
+3. `./pkc --nuke` is a command which deletes the mountpoint (the persistent data) and deletes the current instances of the docker containers.
 
 **Test if deployment was successful:**\
 Go to localhost:9352 and see if you can open your 'Personal Knowledge Container'\
 If the special pages work, congratulations! You successfully deployed PKC!
+## POST Installation (installation of remote verification tools)
 
-## POST Installation (installation of remote verification tools):
-* You have the choice to use the commandline verification tool or the online-chrome extension (recommanded) to verify your page.
+* You have the choice to use the command line verification tool or the online-chrome extension (recommended) to verify your page.
 * Visit https://github.com/inblockio/data-accounting-external-verifier for hte command line tool.
 * Visit https://github.com/inblockio/VerifyPage for the chrome extension.
 
 Keep in mind that this software is in alpha-release stage. Please report bugs and issues back to us.
-
 ## Repository Dependencies Github
+
 *Those repositories are automatically installed by the `pkc` CLI during the setup. 
 - MediaWiki extension https://github.com/inblockio/DataAccounting
   This contains all scripts and information for the 'Verified Page History' implementation.
   languages: PHP, JavaScript, Shell
 - Dockerized PKC https://github.com/inblockio/micro-PKC
-  For running the PKC MediaWiki including the DataAccounting extension via docker-compose.
+  For running the PKC MediaWiki including the DataAccounting extension via Docker Compose.
   languages: Shell, PHP, JavaScript (and Docker of course)
 - Content for population of a fresh installed pkc https://github.com/inblockio/PKC-Content
   This contains all the content which is pulled into the PKC through the create of an initial set of pages.
 - Smart Contract for Witnessing / Timestamping https://github.com/inblockio/DataSymmetry
   This Smart Contract is part of the infrastructure required to interact with the Ethereum network for time-stamping 
-  and 'proof of existance' of a set of verified page fiels.  
+  and 'proof of existence' of a set of verified page fields.  
```

Comment:
Did you use `codespell` for linting the spellings? We should add `codespell` to CI, just like in the PKC-Content repo, i.e. https://github.com/inblockio/PKC-Content/blob/cf365475cb5c5f75f18e87fbd3fa867bf4586182/.github/workflows/ci.yml#L9-L17.

Reply:
I've made a note and will take a look at that when I'm working on CI hardening. The downside to adding it to CI is that we'd then be in the business of maintaining an ever-growing dictionary, which would be more work than fixing misspellings in the editor while modifying files (using something like codespell, for example).